### DISCUSSION TITLE
"indentation_type" should replace all tabs

### DISCUSSION THREAD

**Discussion by mvorisek** [PHP-CS-Fixer/PHP-CS-Fixer#7802] [Feature ideas]
## Feature request

repro file: https://github.com/mvorisek/PHP-CS-Fixer/tree/repro_unfixed_tabs (in last/HEAD commit)

Currently "indentation_type" rule seems to fix indentation purely on the line start.

But I would expect all tabs to be fixed to spaces.

### example in code:

![image](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/2228672/2784cff8-db57-48c5-9998-d5bee4a0c177)

### example in comments:

![image](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/2228672/51b3d054-6a93-4821-aac3-9a58a87bdd62)

**Comment by Wirone** [comment]
I am not sure about this. _Indentation_ has explicit meaning, it's about nesting consecutive parts of the code. Whitespace between `}` and comment is _not_ indentation IMHO. Same goes for phpDoc. In my opinion both cases given as example here are not in the scope of `indentation_type` fixer.

**Comment by mvorisek** [comment]
This is a feature request "to fix all tabs", feel free to edit the title.

**Reply by Wirone** [reply]
I've moved it to "feature ideas" because none of core team members is going to work on this. Feel free to provide implementation (same for anyone interested in this), but it does not make sense to keep it in the main backlog.

**Comment by repressedRadar** [comment]
help!

**Comment by repressedRadar** [comment]
we can't use your extension because of the no tabs issue!!!  and yet everyone says how great it is?  Please help!

**Reply by keradus** [reply]
I do not understand this msg. What extension? What issue? please rephrase and give examples.
